src/Integration/Aws/DynamoDbSession/.github/workflows/branch_alias.yml

```diff
@@ -0,0 +1,76 @@
+name: Update branch alias
+
+on:
+  push:
+    tags: ['*']
+
+jobs:
+  branch-alias:
+    name: Update branch alias
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Set up PHP
+        uses: shivammathur/setup-php@2.5.0
+        with:
+          php-version: 7.4
+          coverage: none
+
+      - name: Find branch alias
+        id: find_alias
+        run: |
+          TAG=$(echo $GITHUB_REF | cut -d'/' -f 3)
```

Comment:
this doesn't allow for a tag with a `/` in it, (unlikely, i know)
we use 
`${GITHUB_REF#refs/tags/}`

Reply:
Cool. I didnt know of that syntax. 

Feel free to send a PR to update this. I usually copy the actions from this repo time to time, so updating might benefit other repos. =)

```
~
❯ GITHUB_REF="refs/tags/0.1.1"

~
❯ echo $GITHUB_REF
refs/tags/0.1.1

~
❯  echo $GITHUB_REF | cut -d'/' -f3
0.1.1

~
❯ echo ${GITHUB_REF#refs/tags/}
0.1.1
```